docs/monitoring-usage.rst

```diff
@@ -154,6 +154,85 @@ before you call
     projects.metricDescriptors
 
 
+Groups
+------
+
+A group is a dynamic collection of *monitored resources* whose membership is
+defined by a `filter`_.  These groups are usually created via the `Stackdriver
+dashboard`_. You can list all the groups in a project with the
+:meth:`~gcloud.monitoring.client.Client.list_groups` method::
+
+    >>> for group in client.list_groups():
+    ...     print(group.id, group.display_name, group.parent_id)
+    ('1001', 'Production', '')
+    ('1002', 'Front-end', '1001')
+    ('1003', 'Back-end', '1001')
+
+See :class:`~gcloud.monitoring.group.Group` and the API documentation for
+`Groups`_ and `Group members`_ for more information.
+
+You can get a specific group based on it's ID as follows::
+
+    >>> group = client.fetch_group('1001')
```

Comment:
``fetch_resource_descriptor`` and ``fetch_metric_descriptor`` exist in the monitoring API Client class.